exaudfclient/base/javacontainer/script_options/parser_legacy.h

```diff
@@ -0,0 +1,53 @@
+#ifndef SCRIPTOPTIONLINEPARSERLEGACY_H
+#define SCRIPTOPTIONLINEPARSERLEGACY_H 1
+
+
+#include "base/javacontainer/script_options/parser.h"
+
+
+namespace SWIGVMContainers {
+
+namespace JavaScriptOptions {
+
+class ScriptOptionLinesParserLegacy : public ScriptOptionsParser {
+
+    public:
+        ScriptOptionLinesParserLegacy();
+
+        void findExternalJarPaths(std::string & src_scriptCode,
```

Comment:
Might it actually better, if the script code goes only once in the parser, and these functions are only for extracting the information? This way, we could parse in the new parser the code once and then only work on the extracted information.